.github/workflows/charm-update-libs.yaml

```diff
@@ -27,26 +27,86 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Fetch charm libraries
+      - name: Merge any PRs from the automatically generated "chore/auto-libs" branch if the CI is green
+        run: |
+          # If a PR from "chore/auto-libs" is open and CI checks are passing, merge it
+          is_pr_open="$(gh pr list --head chore/auto-libs --state open --json id --jq 'length')"
+          if [[ "$is_pr_open" == "1" ]]; then
```

Comment:
Just to be defensive, should we fail loudly if the length is > 1?

Reply:
Done!